airbyte-config/config-persistence/src/main/java/io/airbyte/config/persistence/ConfigWriter.java

```diff
@@ -105,6 +105,8 @@ static void writeStandardSourceDefinition(final List<StandardSourceDefinition> c
             .set(Tables.ACTOR_DEFINITION.RESOURCE_REQUIREMENTS,
                 standardSourceDefinition.getResourceRequirements() == null ? null
                     : JSONB.valueOf(Jsons.serialize(standardSourceDefinition.getResourceRequirements())))
+            .set(ACTOR_DEFINITION.ALLOWED_HOSTS, standardSourceDefinition.getAllowedHosts() == null ? null
```

Comment:
Not exactly sure what the difference is, but I see other entries here are doing this: should this be `Tables.ACTOR_DEFINITION`?

Reply:
Might as well match! https://github.com/airbytehq/airbyte/pull/21363/commits/cac0426da7b9187caecab9a1552f92b13ab4db9f